MoreLinq.Test/ScanTest.cs

```diff
@@ -54,6 +63,8 @@ public void SeededScanSum()
         {
             var result = Enumerable.Range(1, 10).Scan(0, SampleData.Plus);
             var gold = new[] { 0, 1, 3, 6, 10, 15, 21, 28, 36, 45, 55 };
+            Assert.Throws<InvalidOperationException>(() =>
+                Enumerable.Range(1, 3).Concat(new BreakingSequence<int>()).Scan(SampleData.Plus).Consume());
```

Comment:
What's the intention here? It doesn't use `result` and a different overload of `Scan` is being called. Seems like a copy & paste error from [line 51](https://github.com/morelinq/MoreLINQ/pull/393/files#diff-071fe005ca9d2e65c1d052c88657c410R51)?